Kickstarter-iOS/Views/Controllers/PledgeDescriptionViewController.swift

```diff
@@ -344,100 +191,46 @@ private let learnMoreTextViewStyle: TextViewStyle = { (textView: UITextView) ->
-  }
-
-  func snapshotData(withContainerView view: UIView) -> RewardPledgeTransitionSnapshotData? {
-    guard
-      let snapshotView = self.rewardCardContainerView.snapshotView(afterScreenUpdates: false),
-      let sourceFrame = self.rewardCardContainerView.superview?
-      .convert(self.rewardCardContainerView.frame, to: view)
-    else { return nil }
-
-    let maskFrame = CGRect(origin: .zero, size: self.rewardCardContainerShadowView.frame.size)
-
-    return (snapshotView, sourceFrame, maskFrame)
-  }
-
-  func destinationFrameData(withContainerView view: UIView) -> RewardPledgeTransitionDestinationFrameData? {
-    guard
-      let containerFrame = self.rewardCardContainerView.superview?
-      .convert(self.rewardCardContainerView.frame, to: view)
-    else { return nil }
-
-    let mask = self.rewardCardContainerShadowView.bounds
-
-    return (containerFrame, mask)
-  }
-
-  func endTransition(_: UINavigationController.Operation) {
-    self.setThumbnailHidden(false)
-  }
+private let rewardTitleLabelStyle: LabelStyle = { (label: UILabel) in
+  label
+    |> \.numberOfLines .~ 2
+    |> \.textColor .~ UIColor.ksr_soft_black
+    |> \.font .~ UIFont.ksr_headline().bolded
+    |> \.adjustsFontForContentSizeCategory .~ true
 }
 
-// MARK: - UIGestureRecognizerDelegate
-
-extension PledgeDescriptionViewController: UIGestureRecognizerDelegate {
-  func gestureRecognizer(
-    _ gestureRecognizer: UIGestureRecognizer,
-    shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer
-  ) -> Bool {
-    if gestureRecognizer is UILongPressGestureRecognizer, otherGestureRecognizer is UITapGestureRecognizer {
-      return true
-    }
-
-    return false
-  }
+private let rootStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
+  stackView
+    |> checkoutSubStackViewStyle
+    |> verticalStackViewStyle
+    |> \.alignment .~ UIStackView.Alignment.top
 }
 
-// MARK: - Reward card thumbnail size calculations
-
-private func rewardCardThumbnailViewSize(
-  with cardContainerViewSize: CGSize,
-  parentWidth: CGFloat
-) -> CGSize {
-  let width = cardContainerViewSize.width
-  let height = cardContainerViewSize.height
-
-  let maxWidth = parentWidth / 4.5
-  let aspectRatio = height / width
+private func attributedLearnMoreText() -> NSAttributedString? {
+  guard let trustLink = HelpType.trust.url(
+    withBaseUrl: AppEnvironment.current.apiService.serverConfig.webBaseUrl
+  )?.absoluteString else { return nil }
 
-  let newWidth = min(maxWidth, max(width / 3, Layout.RewardThumbnail.minWidth))
-  let newHeight = newWidth * aspectRatio
+  let storeString = Strings.Kickstarter_is_not_a_store_Its_a_way_to_bring_creative_projects_to_life()
+  let accountabilityString = Strings.Learn_more_about_accountability()
+  let fullString = storeString + " " + accountabilityString
 
-  return .init(width: newWidth, height: newHeight)
-}
+  guard let attributedString = checkoutAttributedLink(with: fullString) else { return nil }
```

Comment:
I don't think we still want to call `checkoutAttributedLink` here because that function is expecting an HTML string.

Where do we set the attributes for the link text, colour etc?